.postcssrc.yml

```diff
@@ -0,0 +1 @@
+postcss: {}
```

Comment:
Do we really need this file?

Reply:
Unfortunately yes :(
For some reason assets compiling does not work on Linux (both CI and Heroku). 
Locally on my Mac evening looks fine. Don’t know the reason yet. I will add a task to backlog. Maybe at some point I will find other solution